src/dependency.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable max-classes-per-file */
+
+import { ReplaceSource } from "webpack-sources";
+
+// eslint-disable-next-line
+// @ts-ignore: What's the right way to refer to this one?
```

Comment:
If you want you can check how I dealed with typings https://github.com/hipstersmoothie/react-docgen-typescript-plugin/pull/32/files#diff-0be3b522a88ca90bbe9a426fa3834d5aedf3998519ad62e107db05fe18d2eac6

Reply:
Cool, thanks for the tip!